config/secrets.yml

```diff
@@ -10,16 +10,91 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
+default: &default
```

Comment:
`default` is more apt, and maps better to rails way of using `default`, like in the case of `database.yml`